docs/examples/video-tutorials.rst

```diff
@@ -0,0 +1,232 @@
+.. _examples-neptune-video-tutorials:
+
+Video Tutorials
```

Comment:
There is quite a lot of videos - difficult to browse it / find something relevant.

Maybe some toc at the top?

Reply:
Figured I first want to see if people read it. If true then I would like to structure this thing more than just a toc